source/extensions/filters/network/thrift_proxy/decoder.h

```diff
@@ -196,6 +196,13 @@ class DecoderCallbacks {
    * @return True if payload passthrough is enabled and is supported by filter chain.
    */
   virtual bool passthroughEnabled() const PURE;
+
+  /**
+   * @return True if the expected message comes from the client.
+   *
+   * See https://github.com/apache/thrift/blob/master/lib/ts/thrift.d.ts#L68.
+   */
+  virtual bool fromClient() const PURE;
```

Comment:
Maybe `isRequest()`? I don't feel strongly, but I think that's the terminology I originally used for calls and oneways.

Reply:
Yeah, that makes sense.